heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -163,17 +165,22 @@ public boolean isEmpty() {
             final AggregationInstance aggregation = request.getAggregation();
             final DateRange range = request.getRange();
             final QueryContext queryContext = request.getContext();
+            final Features features = request.getFeatures();
 
             queryLogger.logIncomingRequestAtNode(queryContext, request);
 
-            final boolean slicedFetch = request.getFeatures().hasFeature(Feature.SLICED_DATA_FETCH);
+            final boolean slicedFetch = features.hasFeature(Feature.SLICED_DATA_FETCH);
+
+            final BucketStrategy bucketStrategy =
+                options.getBucketStrategy().orElse(BucketStrategy.END);
```

Comment:
Yes I understand. It's just that Feature.END_BUCKET is unused as the code is right now. I would have expected it to be hooked up, just for completeness of the commit, even if the !hasFeature path would be just an exception.
No biggie tho of course :)